tests/observability/tests/crd_status.go

```diff
@@ -47,13 +67,13 @@ var _ = Describe(tsparams.TnfCrdStatusTcName, Serial, func() {
 		By("Start TNF " + tsparams.TnfCrdStatusTcName + " test case")
 		err = globalhelper.LaunchTests(tsparams.TnfCrdStatusTcName,
 			globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()),
-			globalhelper.GetConfiguration().General.TnfReportDir,
```

Comment:
Is `globalhelper.GetConfiguration().General` used by anyone? If not, it should be removed.

Reply:
Good question, maybe another PR to address that one.